src/rules/map-keys-quotes/__tests__/index.js

```diff
@@ -11,6 +11,40 @@ testRule(rule, {
         $test: ("foo": 14px, "bar": 25px);
       `,
       description: "accepts strings without quotes"
+    },
+    {
+      code: `
+        $test: ('foo': 14px, 'bar': 25px);
+      `,
+      description: "accepts strings without quotes"
```

Comment:
```suggestion
      description: "accepts strings with quotes"
```

?

Reply:
Yeah I copy/pasted it from the other test :)